packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -256,8 +256,9 @@ class Editor extends React.Component {
         font-size: ${theme.typography.body1.fontSize};
 
         /* 80px - 2 * 1px borders  */
+        max-height: 300px;
```

Comment:
Can we actually make this a prop? Would be useful if we need to use this component in different places.

I would suggest a default value of just "auto".

Reply:
@jordanell so the prop would just be optional, and it defaults to auto height if that prop isn't provided?

Reply:
@imogenem that sounds pretty good to me!